Podfile.lock

```diff
@@ -24,6 +24,6 @@ SPEC CHECKSUMS:
   AEXML: 6fc6433aa35bdc15dd8eb8d54eb7aa4520a010eb
   Alamofire: fef59f00388f267e52d9b432aa5d93dc97190f14
 
-PODFILE CHECKSUM: cbb8fb5bea0acb75f752f3138f1de27b691930ff
+PODFILE CHECKSUM: a5aa94a95f8c1cee7c46348bfbc0ba1bbe4b2fc7
 
-COCOAPODS: 1.1.0.rc.2
+COCOAPODS: 1.0.0
```

Comment:
You should use the release candidate of CocoaPods. There is some issue with Swift 3 and the prerelease handles it more properly.